src/lib/save-assets.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const log = require('./log.js');
+const stringify = require('json-stringify-safe');
+
+function getFilenamePrefix(options) {
+  const date = options.date || new Date();
```

Comment:
> would it be better to just be able to specify a prefix?

I think we'll always want date timestamps on these.. Kinda don't want the next layer of configurability. Maybe i'm misunderstanding tho